runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -2247,6 +2260,33 @@ void init_ofiEp(void) {
   tciTabLen = numTxCtxs;
   CHPL_CALLOC(tciTab, tciTabLen);
 
+  // Use "hybrid" MR mode for the cxi provider if it's available
+
+  if (providerInUse(provType_cxi)) {    
```

Comment:
Let's add an env var so we we can experiment with this without rebuilding. Say `CHPL_RT_COMM_OFI_HYBRID_MR`  or if we want it to be cxi specific maybe `CHPL_RT_COMM_OFI_CXI_HYBRID_MR`